src/ts/components/errormessageoverlay.ts

```diff
@@ -99,16 +102,18 @@ export class ErrorMessageOverlay extends Container<ErrorMessageOverlayConfig> {
     let config = <ErrorMessageOverlayConfig>this.getConfig();
 
     player.on(player.exports.Event.Error, (event: ErrorEvent) => {
-      let message = event.message;
+      let message = ErrorEventUtil.defaultErrorMessageTranslator(event);
 
+      // errorMessages configured in `UIConfig` take precedence `ErrorMessageOverlayConfig`
+      let customErrorMessages = uimanager.getConfig().errorMessages || config.messages;
```

Comment:
Call it `errorMessages`. They are only "custom" when provided via the config, not by default.